packages/frontend/src/components/LoginForm.tsx

```diff
@@ -145,15 +148,15 @@ export default function LoginForm({ credentials, setCredentials }: LoginProps) {
             </div>
           )}
 
-          <p className='text'>{tx('login_no_servers_hint')}</p>
+          {!isChatmail && <p className='text'>{tx('login_advanced_hint')}</p>}
```

Comment:
thinking it over, that hint makes sense for configured accounts as well, even if chatmail currently, they are advanced, should not changed to shared, and if changed, may become unencrypted. cmp. https://github.com/deltachat/deltachat-android/pull/4005#issuecomment-3504267145

```suggestion
          <p className='text'>{tx('login_advanced_hint')}</p>
```